src/flutter/flutter-wizard.ts

```diff
@@ -0,0 +1,119 @@
+import { WizardOptions } from '../utils/types';
+import * as Sentry from '@sentry/node';
+import * as codetools from './code-tools';
+import * as fs from 'fs';
+
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import * as clack from '@clack/prompts';
+import chalk from 'chalk';
+
+import {
+  confirmContinueIfNoOrDirtyGitRepo,
+  getOrAskForProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+
+import { traceStep, withTelemetry } from '../telemetry';
+import { findFile } from './code-tools';
+
+export async function runFlutterWizard(options: WizardOptions): Promise<void> {
+  return withTelemetry(
+    {
+      enabled: options.telemetryEnabled,
+      integration: 'android',
+      wizardOptions: options,
+    },
+    () => runFlutterWizardWithTelemetry(options),
+  );
+}
+
+async function runFlutterWizardWithTelemetry(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Flutter Wizard',
+    promoCode: options.promoCode,
+  });
+
+  await confirmContinueIfNoOrDirtyGitRepo();
+
+  const { selectedProject, selfHosted, sentryUrl, authToken } =
+    await getOrAskForProjectData(options, 'flutter');
+
+  const projectDir = process.cwd();
+  const pubspecFile = findFile(projectDir, 'pubspec.yaml');
```

Comment:
We should not look for this file, if the user is in a directory without `pubspec.yaml`, likely they are not in root of the project they want to patch.

We should handle the missing `pubspecFile` and let user know they are not in Flutter project dir.